common/app/conf/switches.scala

```diff
@@ -572,6 +572,24 @@ object Switches {
     exposeClientSide = true
   )
 
+  val TopAboveNavAdSlot728x90Switch = Switch(
+    "Commercial",
+    "top-above-nav-728-90",
+    "Expect a 728 x 90 ad in top-above-nav slot on UK network front.",
+    safeState = Off,
+    sellByDate = new LocalDate(2015, 7, 22),
+    exposeClientSide = true
```

Comment:
I find it quite useful to be able to check their state on the clientside so that I know that the page I'm looking at is using the switch in the state I expect it to be in.